pkg/wal/processor/transformer/wal_transformer_validator.go

```diff
@@ -125,6 +125,8 @@ func pgTypeCompatibleWithTransformerType(compatibleTypes []transformers.Supporte
 		return slices.Contains(compatibleTypes, transformers.DateDataType)
 	case pgtype.TimestampOID, pgtype.TimestamptzOID:
 		return slices.Contains(compatibleTypes, transformers.DatetimeDataType)
+	case pgtype.JSONBOID:
+		return slices.Contains(compatibleTypes, transformers.JSONDataType)
```

Comment:
I wonder what other types we might want this to be used with. Theoretically, just like jsonb, I'd expect it to work well with any pg type that can be represented as a string.

Reply:
Yeah, this new transformer will only be compatible with strings, json and byte arrays, shouldn't we cast a much wider net adding all types that can be represented as strings? Arrays come to mind as well.